ethkey/src/extended.rs

```diff
@@ -21,6 +21,46 @@ use Public;
 use bigint::hash::{H256, FixedHash};
 pub use self::derivation::Error as DerivationError;
 
+pub trait Label {
```

Comment:
particularly the implied invariant on `store` that `target.len()` must be `> T::len()`, and that breaking this invariant will lead to panic.